cellprofiler/modules/exporttodatabase.py

```diff
@@ -1405,7 +1406,9 @@ def add_image_group(self, can_remove=True):
                 doc="""\
 *(Used only if creating a properties file and specifying the image information)*
 
-Enter a color to display this channel.""",
+Enter a color to display this channel.
+
+Multichannel images will use this colour for all 3 image components""",
```

Comment:
Consistent spelling please :)

Reply:
(sorry, was written mostly by damned dirty americans ;) )

Reply:
Is fixed